lib/systemFlake.nix

```
@@ -141,6 +156,10 @@ let
                 networking.hostName = hostname;
               })
 
+              (optionalAttrs (options ? networking.domain) {
+                networking.domain = domain;
+              })
+
```

```suggestion
                networking.hostName = lib.mkDefault hostname;
              })

              (optionalAttrs (options ? networking.domain) {
                networking.domain = lib.mkDefault domain;
              })

```
In case you don't want to pass domains to fup.

---

Good point! However, I guess the correct implementation I guess is slightly different, I'll push in a few seconds.

---

got it, yeah that looks right